From 67b959c21b6f12f8b037401beb43a6efe8278c7c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 11 Dec 2024 16:19:05 +0800
Subject: [PATCH 264/301] AOSCOS: staging: sb105x: drop TTY_DRIVER_MAGIC
 assignment

Per commit 5052df99d3bc ("tty: remove TTY_DRIVER_MAGIC"), the magic number
"shoudl not need" to be defined. Let's see how it goes.

Fixes: "AOSCOS: Revert "staging: sb105x: delete the driver""
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/staging/sb105x/sb_pci_mp.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/staging/sb105x/sb_pci_mp.c b/drivers/staging/sb105x/sb_pci_mp.c
index 32f60de269cc..841e9b15486b 100644
--- a/drivers/staging/sb105x/sb_pci_mp.c
+++ b/drivers/staging/sb105x/sb_pci_mp.c
@@ -1736,7 +1736,6 @@ static int mp_register_driver(struct uart_driver *drv)
 	drv->tty_driver = normal;
 
 	normal->owner           = drv->owner;
-	normal->magic		= TTY_DRIVER_MAGIC;
 	normal->driver_name     = drv->driver_name;
 	normal->name		= drv->dev_name;
 	normal->major		= drv->major;
-- 
2.48.1

